packages/interbit/src/chainManagement/createChains.js

```diff
@@ -38,13 +38,12 @@ const createChainsFromManifest = async (location, cli, manifest, options) => {
   const genesisBlockEntries = Object.values(genesisBlocks)
   for (const genesisBlock of genesisBlockEntries) {
     const chainId = genesisBlock.blockHash
-    if (options.connect) {
+    try {
       await cli.loadChain(chainId)
       console.log(`Loaded chain ${chainId}`)
-    } else {
+    } catch (e) {
       await cli.startChain({ genesisBlock })
```

Comment:
This catches any exception. It would be better to verify that the exception is what we would expect if the chain does not exist. If so, carry on to start the chain, otherwise rethrow.